test/testdata/simpletests/datavolume2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Name": "DataVolume2",
+  "Description": "Verify that more complex datavolumes (including empty and volumes-from) work as expected; see Related",
+  "TaskDefinition": "datavolume2",
+  "Version": ">1.0.0",
+  "Timeout": "2m",
+  "ExitCodes": {
+    "exit": 42
+  },
+  "Related": ["forums#176057", "gh#52"]
```

Comment:
This feels much more like a comment to me.  I think it's fine for now, but we should think about other alternatives.  Maybe change these to full links in the interim rather than knowing forums -> ECS forums and gh -> GitHub.